src/commands/Minion/open.ts

```diff
@@ -74,7 +74,8 @@ export default class extends BotCommand {
 
 		await msg.author.removeItemFromBank(clueTier.id, quantity);
 
-		let loot = clueTier.table.open(quantity * rand(1, 3));
+		const newQuantity = Math.floor(Math.random() * (quantity * 3 - quantity + 1) + quantity);
```

Comment:
to make this accurate to what it used to be consider using a for loop for each casket case. 

something along the lines of 
for (let i = 0; i < quantity; i++ {
  //Roll 1 to 3
  //Add and track extra rolls
 //Combine loot for each cycle.
}